.github/workflows/swift-toolchain.yml

```diff
@@ -263,20 +263,8 @@ on:
 env:
   SCCACHE_DIRECT: on
 
-  # Workaround for needing llvm-17 on macOS preventing us from using the 5.10 toolchain release.
-  WORKAROUND_MACOS_PINNED_BOOTSTRAP_TOOLCHAIN_BRANCH: swift-6.0.1-release
-  WORKAROUND_MACOS_PINNED_BOOTSTRAP_TOOLCHAIN_TAG: 6.0.1-RELEASE
```

Comment:
We still need that one

Reply:
Do you mean we need it for the macos case? Would keeping that conditionally for macos while updating this way for windows work? @Steelskin

Reply:
We don't have a mac toolchain build from `thebrowsercompany` so we can't use the same build. We do need the separate bootstrap toolchain from upstream for macOS. We can definitely update the Windows toolchain.

Reply:
Done